gnocchi/cli.py

```diff
@@ -86,7 +86,11 @@ def change_sack_size():
     ])
     conf = service.prepare_service(conf=conf)
     s = storage.get_incoming_driver(conf.incoming)
-    report = s.measures_report(details=False)
+    try:
+        report = s.measures_report(details=False)
+    except Exception:
+        # issue is already logged by measures_report
```

Comment:
here: https://github.com/gnocchixyz/gnocchi/blob/master/gnocchi/storage/incoming/_carbonara.py#L47-L48

basically what's happening is if we didn't upgrade, it can't find NUM_SACKS value. all the drivers will return back None here https://github.com/gnocchixyz/gnocchi/blob/master/gnocchi/storage/incoming/_carbonara.py#L45 which causes int(None) failure.

i'm not sure what you want me to catch here or do you want me to raise an error instead of having drivers return None? and if so, do you want me to still log here: https://github.com/gnocchixyz/gnocchi/blob/master/gnocchi/storage/incoming/_carbonara.py#L45 or let it raise and have to catch it everywhere it's used?